src/makieplotting.jl

```diff
@@ -172,9 +172,10 @@ function Makie.plot!(WF::Wireframe{<:Tuple{<:MakiePlotter{dim}}}) where dim
     # u_matrix = @lift($(WF[:deformation_field])===:default ? zeros(0,3) : transfer_solution(plotter; field_idx=Ferrite.find_field(plotter.dh,$(WF[:deformation_field])), process=identity))
     # coords = @lift($(WF[:deformation_field])===:default ? plotter.physical_coords : plotter.physical_coords .+ ($(WF[:scale]) .* $(u_matrix)))
     #original representation
+    pointtype = dim > 2 ? Point3f : Point2f
```

Comment:
Maybe we can be more specific regarding the dimension, so we can e.g. error in the case dim = 1 or dim > 3.

Reply:
or get the type via some dispatch

Reply:
But there are only these two types. What's the benefit of a dispatch at this case?

Reply:
Benefit of dispatch over having the error management via if/else? None. Benefit over having no error management? Clearer error messages for users.

Reply:
And on what you want to dispatch? A Val? It's unclear to me and Jan how it should be changed

Reply:
Maybe something like
```suggestion
    if dim == 2
        pointtype = Point2f
    elseif dim == 3
        pointtype = Point3f
    else
       @error "Dimension $dim not supported for wireframe"
    end
```
or 
```suggestion
getpointtype(::Val{3}) = Point3f
...
```
not sure what works best here. Having dispatches here reduces code duplication (we have such ifs at several points in the code). Also keeping https://github.com/Ferrite-FEM/FerriteViz.jl/issues/41 in the back of my head here.

Reply:
Yes #41 needs 2D as well right, that's why we wrote it initially like this, but then let's do the dispatch approach @JanM12

Reply:
Depends on your use case. I think it is also nice to have potting routines for pure 1D problems.

Reply:
Yeah but they need to be plotted on a 2D surface right :D x for the domain and y for the data I guess